src/undo-redo.js

```diff
@@ -118,26 +116,19 @@ function undoDeleteNode(data){
       	group.attr("children_ids", group.attr("children_ids").split(' ').filter(id => id !== node_id).join(' ') );
```

Comment:
Shouldn't this be adding the id?  Also, go ahead and make this a helper function since its done repeatedly